### ISSUE TITLE
TiC hotkeys don't respect key 0 (Esc) meaning "NONE" and react when 'Esc' is pressed

### ISSUE THREAD

**Issue by Barhandar** [SlimeKnights/TinkersConstruct#1324]
Other mods with hotkeys do not react when set to NONE and Esc is pressed.

**Comment by Adaptivity** [comment]
Can you rephrase exactly what you mean? They are respect Esc.

**Comment by TherminatorX** [comment]
I'm guessing they mean that when the hotkey is set to "none," your goggles still zoom in when Esc is pressed.. Haven't tried this yet, though.

**Comment by Adaptivity** [comment]
If so I can't reproduce it.

**Comment by Barhandar** [comment]
>Can you rephrase exactly what you mean? They are respect Esc. And what TiC version are you using?

Sorry for the highly delayed answer. TiCo 1.8.2.a, if Armor GUI's key is set to NONE, eventually it opens when trying to open ESC game menu and continues to open on attempts to get away from it, only spamming it allows me to get into the menu and switch the key to something else.

**Comment by Adaptivity** [comment]
Still can't reproduce it. Maybe you can make a video or gif of this bug?

**Comment by TherminatorX** [comment]
As a work-around after you post the gif/vid, there is a config that allows you to change hotkeys out-of-game. I forget the name, but it's likely a Forge thing.

**Comment by bonii-xx** [comment]
Can't reproduce.
